python/ml4ir/applications/classification/tests/test_classification_model.py

```diff
@@ -45,7 +45,9 @@ def test_csv(self):
         metrics = self.run_default_pipeline(data_format="csv")
 
         # Check if the loss and accuracy on the test set is the same
-        assert np.isclose(metrics["loss"], 1.966392993927002, rtol=0.01)
-        assert np.isclose(metrics["categorical_accuracy"], 0.18229167, rtol=0.01)
-        assert np.isclose(metrics["Precision"], 0.2, rtol=0.01)
```

Comment:
I removed Precision because it is not relevant for our model.
Basically, what Precision do by default is to consider 'true' a prediction that is above 0.5 (0.5 is the default threshold).
There is a parameter that allows to instead consider the best score (top_k=1), but then the metric will be the same as CategoricalAccuracy I think.